components/01 - elements/accordions/accordions.js

```diff
@@ -0,0 +1,34 @@
+// This would be a unique ID for every accordion.
```

Comment:
Is this comment necessary, since the line below is selecting by class now?

Reply:
Thanks for the catch, this has been removed.